Docs/source/dataanalysis/catalyst.rst

```diff
@@ -0,0 +1,167 @@
+.. _visualization-catalyst:
+
+In situ Visualization with Catalyst 2
+=====================================
+Catalyst 2 (further referred to as just Catalyst) is a lightweight in-situ visualization and analysis framework API developed for simulations and other scientific data producers. It has a lightweight implementation
+(or **stub**) and an SDK to develop custom implementations of Catalyst. ParaView comes with its own implementation (known as **ParaView Catalyst**) for leveraging ParaView's
+visualization and analysis capabilities, which is what this document will focus on.
+
+
+Enabling Catalyst
+-----------------
+In order to use Catalyst with WarpX, you must `build Catalyst 2 <https://catalyst-in-situ.readthedocs.io/en/latest/build_and_install.html>`_ and `build <https://github.com/Kitware/ParaView/blob/master/Documentation/dev/build.md>`__ or `install <https://www.paraview.org/download/>`__ ParaView 5.9+. Afterward, AMReX must be built with ``AMReX_CONDUIT=TRUE``,
+``AMReX_CATALYST=TRUE``, ``Conduit_DIR=/path/to/conduit``, and ``Catalyst_DIR=/path/to/catalyst`` (``/path/to/catalyst`` should be the directory containing ``catalyst-config.cmake``, not the path to the implementation).
+
+Once AMReX is appropriately built, WarpX can be built with the following options:
+
+.. code-block:: cmake
+
+    WarpX_amrex_internal=FALSE
+    AMReX_DIR="/path/to/amrex/build"
```

Comment:
Since we build AMReX as a superbuild by default, I would rather document to pass the `-DAMReX_CONDUIT=ON -DAMReX_CATALYST=ON` flags to a WarpX build.

(We do support external AMReX installs, of course, but do not do so outside of package managers.)